app/src/main/java/com/github/muellerma/mute_reminder/ForegroundService.kt

```diff
@@ -75,16 +78,26 @@ class ForegroundService : Service() {
             shouldNotify -> {
                 Log.d(TAG, "Should notify, show notification")
 
+                //create pendingIntent, click to send mute action
+                val intent = Intent(this, ForegroundService::class.java)
+                intent.action = "mute_media"
```

Comment:
```suggestion
                intent.action = ACTION_MUTE
```
ACTION_MUTE should be a const in the companion object.

Reply:
> ACTION_MUTE should be a const in the companion object.

@du33169

Reply:
Sorry, my bad. Should be okay now.